### ISSUE TITLE
error when trying to install extension from WSL2

### ISSUE THREAD

**Issue by alanlivio** [microsoft/vscode#93121]
I tried to install an extension in WSL2 and I received the following error.

` ` ` 
$ code --install-extension
` ` ` 
` ` ` 
  Ignoring option install-extension: not supported for code.
` ` `

**Comment by aeschli** [comment]
Yes, the code cli in WSL does not support installing extensions.
